libr/debug/dreg.c

```diff
@@ -57,7 +57,7 @@ R_API int r_debug_reg_sync(RDebug *dbg, int type, int write) {
 			int bufsize = dbg->reg->size;
 			//int bufsize = dbg->reg->regset[i].arena->size;
 			if (bufsize > 0) {
-				ut8 *buf = calloc (1, bufsize);
+				ut8 *buf = calloc (1+1, bufsize);
```

Comment:
> why calloc 2?

That's a *good* question... mostly because valgrind told me there were reads pas the end of that buffer when running ```t.archos/Linux-x86_64/db_maps``` tests. Just trying to "you see it you fix it." I kept this change separate in commit  8f403df6313088b5c90ab19603dc41621d4809b9 in case this 'fix' was the wrong one.